packages/round-manager/src/features/round/ViewRoundSettings.tsx

```diff
@@ -2238,14 +2234,17 @@ function Funding(props: {
             </span>
           </div>
           <div
-            className={`leading-8 font-normal ${
+            className={`leading-8 flex font-normal ${
               !props.editMode.canEdit ||
               (props.editMode.canEditOnlyRoundEndDate && "text-grey-400")
             }`}
```

Comment:
This library is great for these instances:
https://github.com/lukeed/clsx

Would look like this:
```tsx
className={clsx(`leading-8 flex font-normal`, {
  ["text-grey-400"]: !props.editMode.canEdit || props.editMode.canEditOnlyRoundEndDate
})
```

Reply:
Indeed, however I would leave this sort of refactoring for the actual refactoring ticket for this page, which is imminent after all the current tickets for this page are implemented.